roles/prometheus-server/templates/prometheus.yml.j2

```diff
@@ -117,22 +117,27 @@ scrape_configs:
 
   - job_name: 'site-up'
     metrics_path: /probe
-    params:
-      module: [http_2xx]
     static_configs:
       - targets:
         - 'https://netsoc.co/'
         - 'https://wiki.netsoc.co/'
         - 'https://netsoc.dev/'
         - 'https://keycloak.netsoc.dev/'
```

Comment:
@jac

Reply:
The /auth/ isn't really necessary as https://keycloak.netsoc.dev/ returns a 200 with a script in the head that sets the page location to /auth/

![](https://snipboard.io/psUqkl.jpg)

It can still be changed though if you would prefer

Reply:
oh i thought it was a 301, ok all good